.dockerignore

```diff
@@ -0,0 +1,6 @@
+Dockerfile
+README.md
+*.pyc
+*.pyo
+*.pyd
+__pycache__
```

Comment:
Can we add `.idea/` also

Reply:
`.idea` should be checked in. It should be part of `.gitignore`